lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
im not entirely sure about this, but i think to use this, you need to create a scope (via a closure of a function )